src/maliput_malidrive/builder/road_curve_factory.h

```diff
@@ -345,6 +359,26 @@ class RoadCurveFactory final : public RoadCurveFactoryBase {
   std::unique_ptr<malidrive::road_curve::Function> MakeCubicFromXodr(
       const std::vector<T>& xodr_data, double p0, double p1, FillingGapPolicy policy,
       road_curve::PiecewiseFunction::ContinuityCheck continuity_check) const;
+
+  /// Constructs CubicPolynomials out of @p lane_widths.
+  ///
+  /// @param lane_widths Contains the lane widths to construct the CubicPolynomials from.
+  /// @param p0 Lower bound extreme of the parameter range. It must not be
+  ///        negative and must be less than @p p1.
+  /// @param p1 Upper bound extreme of the parameter range. It must be greater
+  ///        than @p p0.
+  /// @returns A vector of cubic polynomial functions.
+  ///
+  /// @throws maliput::common::assertion_error When any of @p lane_width's offset is less than one of the priors.
+  std::vector<std::unique_ptr<malidrive::road_curve::Function>> MakeCubicPolynomialsFromLaneWidths(
+      const std::vector<xodr::LaneWidth>& lane_widths, double p0, double p1) const;
+
+  /// Informs whether the input lane widths are C1 continuous.
+  /// @param a Starting lane width.
+  /// @param b Ending lane width.
+  /// @param b_end @p b's end horizontal coordinate.
```

Comment:
This is only used to create the cubic pol but you don't care about that (if it is 1 or if itis 1000 it is the same for the adaptation) I wonder if we could avoid requesting a b_end which is a bit misleading actually.

Reply:
Sure. We can use an arbitrary value for the `b` polynomial's `p1`.